### ISSUE TITLE
Update closure-compiler to v20151216

### ISSUE THREAD

**Issue by cmbankester** [documentcloud/closure-compiler#36]
https://github.com/google/closure-compiler/wiki/Releases#december-16-2015-v20151216

**Comment by reefdog** [comment]
Hah, what timing. Happy to update to this, though right now three of our tests are failing (and have been for a while). I suspect our test data needs updating. @cmbankester would you mind taking a crack at fixing the tests? Might be a while before I can get to it.

**Comment by cmbankester** [comment]
I'm swamped currently, but I'll certainly look into when I have time!

**Comment by reefdog** [comment]
Cool. I won't let that hold this up, but it'd be real nice.

**Comment by YurySolovyov** [comment]
Will (any)one accept a PR with a bump to the latest stable version? /cc @documentcloud

**Comment by knowtheory** [comment]
Yep. This got bumped back down the priority list, so i'm willing to accept a PR even if the tests aren't fixed, so long as someone has run through at least some tests by hand to verify that nothing weird is going on.  If we have to back out the changes that's a better way to go than just letting it linger forever.

@YurySolovyov any chance you'd be able to take a crack at it?

**Comment by YurySolovyov** [comment]
I can promise submitting a PR, but not fixing tests right away, cause, you know, need to get familiar with code first. May need some guidance to get there

**Comment by knowtheory** [comment]
Definitely happy to lend a hand on that front.  The tests are actually pretty dumb/brute force (also ancient).

Most of the failing tests are matters of verifying that the output of the closure-compiler black box was the same as what it was before.

That also means we're not taking any advantage of smarter things one could do w/ the toolkit.

**Comment by YurySolovyov** [comment]
@cmbankester release 1.13 comes with a newer compiler, can you try it out and confirm that desired features are there?
